create/create.js

```diff
@@ -8,24 +8,33 @@ import {
 const form = document.querySelector('.bunny-form');
 const logoutButton = document.getElementById('logout');
 
-form.addEventListener('submit', async e => {
+form.addEventListener('submit', async (e) => {
     // prevent default
-
+    e.preventDefault();
     // get the name and family id from the form
-
+    const data = new FormData(form);
+    const name = data.get('bunny-name');
+    const familyId = data.get('family-id');
     // use createBunny to create a bunny with this name and family id
-    
+    const bunny = { name: name, family_id: familyId };
+    await createBunny(bunny);
     form.reset();
+    window.location.href = '/';
```

Comment:
Since the '/' path reflects your auth form - it might be better to redirect the user to the list view here (/families) instead. Currently your auth page is checking for a user, and redirecting, which works - but to avoid putting the user through that routing, you could update the path here instead.